cla_public/static-src/stylesheets/forms/_form-subfield.scss

```diff
@@ -7,15 +7,6 @@
   zoom: 1;
   overflow: hidden;
 
-  > .form-group-label {
-    float: left;
-    margin-bottom: 10px;
-
-    label:after {
-      content: none;
-    }
-  }
-
   .input-prefix {
     margin: 0 5px 0 0;
     display: inline-block;
```

Comment:
Is this file still needed as it is removed from the forms SASS file

Reply:
The fact that I deleted some of its content makes me think that the other content was still needed.  But there were so many, I can't remember.  I'll look into it.

Reply:
This file should have been deleted.  Done now.